### ISSUE TITLE
wrong urls added to sitemap

### ISSUE THREAD

**Issue by Alex0007** [ekalinin/sitemap.js#27]
```javascript
var sm = require('sitemap')

var staticUrls = ['/', '/terms', '/login']


var sitemap = sm.createSitemap({
    hostname: process.env.HOST_URL || 'http://babeleo.com',
    urls: staticUrls
})

sitemap.add({
    url: '/details/' + 'url1'
})

console.log('sitemap1 urls', sitemap.urls)

var sitemap2 = sm.createSitemap({
    hostname: process.env.HOST_URL || 'http://babeleo.com',
    urls: staticUrls
})

console.log('sitemap2 urls', sitemap2.urls)

```

And the output is:
```
sitemap1 urls [ '/', '/terms', '/login', { url: '/details/url1' } ]
sitemap2 urls [ '/', '/terms', '/login', { url: '/details/url1' } ]
```

Expected output:
```
sitemap1 urls [ '/', '/terms', '/login', { url: '/details/url1' } ]
sitemap2 urls [ '/', '/terms', '/login' ]
```

**Comment by ekalinin** [comment]
Hi @Alex0007, 
Thanks for report.

Will check a bit later.

**Comment by denisbetsi** [comment]
Added a fix in a pull request along with additional  feature

**Comment by ekalinin** [comment]
Fixed in #33

**Comment by Alex0007** [comment]
PR didn't fix this problem

**Comment by denisbetsi** [comment]
I've tested that locally it worked. Would you share your code?

**Comment by Alex0007** [comment]
Code from the top-post
![](http://i.imgur.com/v5n4pyM.png)

**Comment by ekalinin** [comment]
Hey @Alex0007 

Seems, you used an old version of the package where all nearest changes weren't included.
I've just released 0.9.0, and added test for your testcase. Seems all works.
Check it out, please.

**Comment by Alex0007** [comment]
It works! Thanks!
